ui/src/pages/frontsite/Documents/DocumentsSearch/DocumentSearchResultsGrid/DocumentSearchResultsGrid.js

```diff
@@ -3,11 +3,16 @@ import { BookCard } from '@pages/frontsite/components/BookCard';
 
 import { Card, Responsive } from 'semantic-ui-react';
 import { ResultsGrid } from 'react-searchkit';
+import { SeriesCard } from '@pages/frontsite/components/SeriesCard';
 
 export default class DocumentSearchResultsGrid extends Component {
   renderResultsGrid = results => {
-    const cards = results.map(book => {
-      return <BookCard key={book.metadata.pid} data={book} />;
+    const cards = results.map(result => {
+      return result.metadata.$schema.includes('documents/document') ? (
```

Comment:
please, mark here with a comment to change the schema checking to pid_type check. As we plan to have pid object, I think it will be safer than to depend on schema url which is less "static"